package.json

```
@@ -35,8 +35,8 @@
 	},
 	"devDependencies": {
 		"@cloudflare/blindrsa-ts": "0.4.3",
+		"@cloudflare/vitest-pool-workers": "^0.8.0",
```

we use fixed dependency for this project

```suggestion
		"@cloudflare/vitest-pool-workers": "0.8.0",
```